src/validation/custom.validation.ts

```diff
@@ -126,17 +126,17 @@ const checkIdentityDateFields = (dayStr: string = "", monthStr: string = "", yea
   }
 };
 
-export const checkStartDate = (dayStr: string = "", monthStr: string = "", yearStr: string = "") => {
+export const checkDate = (dayStr: string = "", monthStr: string = "", yearStr: string = "") => {
   // to prevent more than 1 error reported on the date fields we first check for multiple empty fields and then check if the year is correct length or missing before doing the date check as a whole.
   if (checkMoreThanOneDateFieldIsNotMissing(dayStr, monthStr, yearStr)
   && isYearEitherMissingOrCorrectLength(yearStr)
   && checkDateIsNotCompletelyEmpty(dayStr, monthStr, yearStr)) {
-    checkStartDateFields(dayStr, monthStr, yearStr);
+    checkAllDateFields(dayStr, monthStr, yearStr);
   }
   return true;
 };
 
-const checkStartDateFields = (dayStr: string = "", monthStr: string = "", yearStr: string = "") => {
+const checkAllDateFields = (dayStr: string = "", monthStr: string = "", yearStr: string = "") => {
```

Comment:
would be nice to add : boolean return type but a bit of a strange one as it throws on error

Reply:
added!

Reply:
ah, just tried to add it to this one but it didn't like it since the method doesn't return true/anything at the end!

Reply:
strange - all paths seem to either throw or return true. But then I'm no compiler. No real need to fix I think.